content/english/javascript-basics/introduction/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Introduction"
+date: 2020-07-28T11:45:38-07:00
+draft: false
+weight: 1
+---
+
+# JavaScript Basics
+ Hi there! If you're reading this, it hopefully means that you're here to learn JavaScript 😁🥳. Whether this is your first time coding or you want a way to quickly pick up a new language - you're in the right place! Get ready to discover the power of programming and have fun. 
+
+ ## What is Code? 
+We can't learn how to code without first understanding what exactly code is. Think of your favorite website, if you can't think of one let's take NuevoFoundation.org. As a user, all you have to do is type the web address in your browser and before you know it - ta da! - the website appears, with all of its pretty colors and pictures of nuvi. 
```

Comment:
Omg, I love this! 😍 This is so good I want to put it on the landing page of all software workshops (which doesn't exist today, but seems like a good idea in my head). Great job!!